### ISSUE TITLE
Using stepsArray with MomentJS instances doesnt

### ISSUE THREAD

**Issue by evenicoulddoit** [angular-slider/angularjs-slider#487]
### Steps to reproduce
1. 
2. 
3. 

Demo: http://jsfiddle.net/cwhgLcjv/ (fork this example and update the link)

### Expected behaviour
Tell us what should happen

### Actual behaviour
Tell us what happens instead

**Comment by ValentinH** [comment]
Indeed, stepsArray should contains values that can be compared with "===". The only exception are Date instances that are handled as well.

**Comment by evenicoulddoit** [comment]
But having not meticulously looked through the code I'm not sure why `===` equality checking would make the above infeasible? The `rz-slider-model` and `rz-slider-high` values are both members of the `stepsArray` - I'm not attempting to compare "similar" moment instances?

**Comment by ValentinH** [comment]
Yes you are right. At first sight, I don't understand what can be the issue and also, I haven't test this slider with ES6 so I don't know if your arrow functions are ok or not.

Anyway, I'm not maintaining this project anymore (or very rarely at least) so I can't help you much.
